packages/ui-kit/src/components/DataGrid/Drawer.tsx

```diff
@@ -0,0 +1,49 @@
+import classNames from 'classnames'
+import React, { Fragment, useRef } from 'react'
+import { Typography } from '../Typography'
+
+import componentStyles from './Drawer.module.scss'
+import { DrawerProps } from './Drawer.types'
+import { ArrowDownIcon, CloseIcon, CopyIcon } from './icons'
+import { getDisplayValue } from './utils'
+
+// TODO: abstract Drawer to a generic component?
+export function Drawer(props: DrawerProps) {
+  const { isOpen, row, cell, onClose, onCsvDownload, ...rest } = props
+
+  const drawerRef = useRef<HTMLDivElement>(null)
+
+  const displayRow = cell ? { id: cell.id, cells: [cell] } : row
+
+  return (
+    <div ref={drawerRef} className={classNames(componentStyles.container, !isOpen && componentStyles.hidden)} {...rest}>
```

Comment:
It's important to include `className` as a last argument in `classNames`, otherwise a custom `className` will completely override the component's styles.

```suggestion
export function Drawer(props: DrawerProps) {
  const { isOpen, row, cell, onClose, onCsvDownload, className, ...rest } = props

  const drawerRef = useRef<HTMLDivElement>(null)

  const displayRow = cell ? { id: cell.id, cells: [cell] } : row

  return (
    <div ref={drawerRef} className={classNames(componentStyles.container, !isOpen && componentStyles.hidden, className)} {...rest}>
```

Reply:
Please also upgrade it to the standard component structure that includes `forwardRef` and `useCombinedRefsCallback`, e.g. [Counter](https://github.com/propeldata/ui-kit/blob/main/packages/ui-kit/src/components/Counter/Counter.tsx#L37)